### PR TITLE
Add area/metering label.

### PR DESCRIPTION
**Pull request by yunkunrao**
Signed-off-by: Rao Yunkun <yunkunrao@yunify.com>

### CODE REVIEW

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Not only this repository `kubesphere/kubesphere` needs this label. For example: `console`, `ks-installer`, `website`.

And I'm wondering if you can post a screenshot to show the color of this label.

**Comment by yunkunrao** [comment]
Currently metering related repos are as follows in my opinion:

- kubesphere/kubesphere
- kubesphere/console
- kubesphere/ks-installer
- kubesphere/community
- kubesphere/website

@LinuxSuRen can we add metering labels for above repos currently, and add more  repos when it's needed? If it's ok, I will update this pr.

**Comment by LinuxSuRen** [comment]
I prefer adding them to this PR.

**Comment by yunkunrao** [comment]
got it

**Comment by yunkunrao** [comment]
> Not only this repository `kubesphere/kubesphere` needs this label. For example: `console`, `ks-installer`, `website`.
> 
> And I'm wondering if you can post a screenshot to show the color of this label.

Yeah, got it.

image

File: ks-ci-bot/labels.yaml [hunk 1]
```diff
@@ -175,6 +175,41 @@ default:
       prowPlugin: label
       addedBy: anyone
 repos:
+  kubesphere/kubesphere:
+    labels:
+      - color: A050cc
```
Comment (LinuxSuRen) [inline ks-ci-bot/labels.yaml]: Please make sure all characters are lower-case.

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Pull request merged**